src/foam/u2/Accordion.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'Accordion',
+  extends: 'foam.u2.Element',
+
+  requires: [ 'foam.u2.ActionView' ],
+
+  css: `
+    ^ {
+      width: 100%;
+      overflow: hidden;
+      background-color: #eee;
+      border: 1px solid #eee;
+      border-bottom-color: #ccc;
+    }
+    ^:first-child {
+      margin-top: 15px;
+      border-radius: 5px 5px 0 0;
+    }
+    ^:last-child {
+      margin-bottom: 15px;
+      border-radius: 0 0 5px 5px;
+      border-bottom-color: #eee;
+    }
+    ^toolbar {
+      background-color: #eee;
+      padding: 5px;
+      display: flex;
+      justify-content: space-between;
+    }
+    ^actions {
+      justify-content: center;
+      align-items: center;
+      display: flex;
+      gap: 5px;
+    }
+    ^actions :is(^action,^toggle) {
+      margin: 0;
+    }
+    ^action {
+      padding: 2px 5px;
+      border: none;
+      background-color: rgba(255, 255, 255, .7);
+    }
+    ^action:hover {
+      background-color: rgb(74, 170, 135) !important;
+      color: white !important;
+    }
+    ^title {
+      padding: 3px;
+      font-size: 1.1em;
+      cursor: default;
+      display: flex;
+      align-items: center;
+    }
+    ^content {
+      background: white;
+      display: block;
+      transition: 0.2s ease-in-out;
+      height: 0;
+      padding: 0;
+      opacity: 0;
+    }
+    ^.expanded ^content {
+      height: auto;
+      padding: 5px;
+      opacity: 1;
+    }
+    ^ .foam-u2-ActionView-toggle svg {
+      width: 12px;
+      height: 12px;
+      color: black;
+    }
+    ^ .foam-u2-ActionView-toggle {
+      transition: transform 0.3s;
+      background: transparent;
+      border-radius: 50%;
+      border: none;
+      outline: none;
+      outline: none;
+      padding: 3px;
+      width: 30px;
+      min-width: 30px;
+      height: 30px;
+    }
+    ^ .foam-u2-ActionView-toggle:hover {
+      background: transparent !important;
+    }
+    ^.expanded .foam-u2-ActionView-toggle {
+      transform: rotate(90deg);
+      transition: transform 0.3s;
+    }
+  `,
+
+  properties: [
+    'title',
+    {
+      name: 'icon',
+      value: null,
+      documentation: `
+        An icon that appears before the title in the Accordion toolbar.
+
+        Example:
+          this.start(foam.u2.Accordion, {
+            title: 'Action Accordion',
+            icon: {
+              class: 'foam.u2.tag.Image',
+              data: 'images/my-icon.svg'
+            },
+          })
+            .add('This is hidden content...');
+      `,
+    },
+    {
+      name: 'actions',
+      value: [],
```

Comment:
Should never do value: [] otherwise all instances will share the same array (unless they set to something else).
Instead do:
factory: function() { return []; }
and then each one will get their own array.
Or, make the class: 'Array' and specify the of: and then it will set the factory for you.